static/app/utils/usePageFilters.tsx

```diff
@@ -0,0 +1,11 @@
+import PageFiltersStore from 'sentry/stores/pageFiltersStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+
+/**
+ * Custom hook that returns the state of page filters
+ */
+function usePageFilters() {
+  return useLegacyStore(PageFiltersStore);
```

Comment:
This will break compatibility with the HoC as this will return `isReady` instead of `isGlobalSelectionReady`, but that seems better since we're no longer calling it global selection anyway.